vendor/github.com/cloudevents/sdk-go/v2/alias.go

```diff
@@ -0,0 +1,173 @@
+// Package v2 reexports a subset of the SDK v2 API.
+package v2
```

Comment:
as you will be removing the experimental file that uses CEs, this dependency will go away for now...
much less files will be in the PR. 
we can add the dependency in a follow up PR, once we introduce the experimental bits